### PR TITLE
feat: add main menu page

### PR DESCRIPTION
**Pull request by AlexeiMet**

### CODE REVIEW

File: src/View/pages/MainMenu/MainMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  public render() {
+    return (
+      <div className='bt-main-menu'>
+        <div className='bt-main-menu__wrapper'>
+          <div className='bt-main-menu__content'>
+            <Switch>
+              <Route exact={true} path='/main-menu/' component={Index} />
+              <Route exact={true} path='/main-menu/options' component={Options} />
+              <Route exact={true} path='/main-menu/players' component={Players} />
+              <Redirect from='/main-menu/' to='/main-menu/' />
```
Comment (vharadkou) [inline src/View/pages/MainMenu/MainMenu.tsx]: wtf?

Comment (MarozauArtsiom) [reply src/View/pages/MainMenu/MainMenu.tsx]: wtf? wtf?
@vharadkou, what do you mean?

Comment (MarozauArtsiom) [reply src/View/pages/MainMenu/MainMenu.tsx]: from='/main-menu/' to='/main-menu/'
destination and origin are the same

**Comment by YegorFeoktistov** [comment]
@AlexeiMet, I think you should merge master branch into your current branch, then make a "Push" again.

**Pull request merged**